### DISCUSSION TITLE
Is qtranslate-xt still maintained?

### DISCUSSION THREAD

**Discussion by nordmathis** [qtranslate/qtranslate-xt#1410] [General]
Hi
qtranslate, qtranslate-x and now qtranslate-xt is the best multilanguage-plugin I ever used. But it seams kind of abandoned. Is there still a team maintaining qtranslate-xt? Is there a roadmap?
Thanks

**Comment by pszurgotradaway** [comment]
Same here... hope someone continues this great work :/

I'm using this plugin since qtranslate version.

**Comment by bagaweb** [comment]
I really hope the project has not been abandoned.

Maybe @herrvigg is busy with another project.

**Comment by rafaelurben** [comment]
I hope so! It's an awesome plugin!

I am afraid that the plugin will break in the near future when WooCommerce enforces its new product editor, as stated in #1407. 🫤

**Comment by rafaelurben** [comment]
Newest update: 

https://github.com/qtranslate/qtranslate-xt/issues/1414#issuecomment-2297627536

> hi,
sorry there has not been any update for such a long time but I'm still around. I've been very busy and it's been difficult times, unfortunately I had no time to dedicate to freetime projects during the last months.
I could do some basic updates if merge requests are easy to validate. The main problem is some changes require a lot of time to understand by analyzing and testing to make sure it doesn't create new regressions... This is a complex project that would need major refactoring and automated tests. Maybe the community could help by testing the merge requests to make it easier to update.

**Comment by herrvigg** [comment]
Yes it is still maintained, with the little time i have now.
Last release 3.15.3 available from 2025-03-29.
https://github.com/qtranslate/qtranslate-xt/releases
